packages/backend-core/src/redis/tests/redis.spec.ts

```diff
@@ -1,8 +1,28 @@
+import { GenericContainer, StartedTestContainer } from "testcontainers"
 import { generator, structures } from "../../../tests"
 import RedisWrapper from "../redis"
+import { env } from "../.."
+
+jest.setTimeout(30000)
 
 describe("redis", () => {
   let redis: RedisWrapper
+  let container: StartedTestContainer
+
+  beforeAll(async () => {
+    const container = await new GenericContainer("redis")
+      .withExposedPorts(6379)
+      .start()
+
+    env._set(
+      "REDIS_URL",
+      `${container.getHost()}:${container.getMappedPort(6379)}`
+    )
+    env._set("MOCK_REDIS", 0)
+    env._set("REDIS_PASSWORD", 0)
```

Comment:
```suggestion
    const cleanup = config.withEnv({
      REDIS_URL: `${container.getHost()}:${container.getMappedPort(6379)}`,
      MOCK_REDIS: 0,
      REDIS_PASSWORD: 0,
    })
```

Then in `afterAll` you can call `cleanup()` and it'll reset the environment variables back to their original values.

Reply:
This is set to package and server. Jest will be restoring these settings between files, so no real need to set this here